decidim-dev/lib/decidim/dev/test/rspec_support/capybara.rb

```diff
@@ -24,7 +24,6 @@ def switch_to_default_host
 
   options = ::Selenium::WebDriver::Chrome::Options.new
   options.args << "--headless"
-  options.args << "--no-sandbox"
```

Comment:
I *think* this was needed for Chrome headless, but I might be wrong 😕

Reply:
I think it was initially needed at some point, but not anymore... See

https://github.com/teamcapybara/capybara/blob/63a12e7ba064fe326b36d014ede8c9269855a6af/lib/capybara.rb#L477-L482

Reply:
Well, the test suite will tell us!

Reply:
Ooops, you were right, indeed!

Reply:
Let me remove that change then!